testing/src/test/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -563,6 +563,35 @@ public void getRowsWithNotFound() throws IOException {
     assertThat(data).isEmpty();
   }
 
+  @Test
+  public void getInvalidWhereClause() throws IOException {
+    createKeyspace(keyspaceName);
+    createTable(keyspaceName, tableName);
+
+    String rowIdentifier = UUID.randomUUID().toString();
+    Map<String, String> row = new HashMap<>();
+    row.put("id", rowIdentifier);
+
+    RestUtils.post(
+        authToken,
+        String.format("%s:8082/v2/keyspaces/%s/%s", host, keyspaceName, tableName),
+        objectMapper.writeValueAsString(row),
+        HttpStatus.SC_CREATED);
+
+    String whereClause = "{\"invalid_field\":{\"$eq\":\"test\"}}";
+    String body =
+        RestUtils.get(
+            authToken,
+            String.format(
+                "%s:8082/v2/keyspaces/%s/%s?where=%s", host, keyspaceName, tableName, whereClause),
+            HttpStatus.SC_INTERNAL_SERVER_ERROR);
```

Comment:
Specifying an invalid field name in query params looks like a user error to me... Do we want to report it as  `SC_INTERNAL_SERVER_ERROR`? :thinking:

Reply:
You right. I am trying to figure out why it's coming as 500 instead of 400.

Reply:
Looks like you caught it down here. See my comment above about throwing something other than a RuntimeException so we can catch and return 400 rather than 500.